travis_jenkins.py

```diff
@@ -267,6 +268,7 @@ def wait_for_building(name, number):
 CATKIN_PARALLEL_TEST_JOBS = %(CATKIN_PARALLEL_TEST_JOBS)s
 BUILD_PKGS       = %(BUILD_PKGS)s
 DOCKER_CONTAINER_NAME   = %(DOCKER_CONTAINER_NAME)s
+JENKINS_HOME            = %(JENKINS_HOME)s
```

Comment:
@k-okada 
Could you please review this why this happen?
https://s3.amazonaws.com/archive.travis-ci.org/jobs/98149483/log.txt
```
JENKINS_HOME            = None

next build number is 1367
wait for trusty-travis-jsk-ros-pkg-jsk_common-indigo-deb-true 1367
start building, wait for result....
http://jenkins.jsk.imi.i.u-tokyo.ac.jp:8080/job/trusty-travis-jsk-ros-pkg-jsk_common-indigo-deb-true/1367/ building.. True result... None
http://jenkins.jsk.imi.i.u-tokyo.ac.jp:8080/job/trusty-travis-jsk-ros-pkg-jsk_common-indigo-deb-true/1367/ building.. True result... None
http://jenkins.jsk.imi.i.u-tokyo.ac.jp:8080/job/trusty-travis-jsk-ros-pkg-jsk_common-indigo-deb-true/1367/ building.. True result... None
Started by user k-okada
Building on master in workspace /var/lib/jenkins/jobs/trusty-travis-jsk-ros-pkg-jsk_common-indigo-deb-true/workspace
[workspace] $ /bin/sh -xe /tmp/hudson1311767791672845649.sh
+ set -x
+ set -e
+ env
JENKINS_HOME=/var/lib/jenkins
rvm_tmp_path=/usr/share/ruby-rvm/tmp
MAIL=/var/mail/jenkins
rvm_rubies_path=/usr/share/ruby-rvm/rubies
rvm_wrappers_path=/usr/share/ruby-rvm/wrappers
USER=jenkins
HUDSON_URL=http://jenkins.jsk.imi.i.u-tokyo.ac.jp:8080/
SHLVL=1
NODE_LABELS=master
HOME=/var/lib/jenkins
```

```
+ source None/.token

/bin/bash: line 226: None/.token: No such file or directory

+ echo 'GitHub Token not found. skipping'
```